qbittorrent/riscv64.patch

```diff
@@ -21,3 +20,21 @@
  build() {
    cmake -B build -S $pkgbase-$pkgver \
      -DCMAKE_INSTALL_PREFIX=/usr \
+-    -DQT6=ON
+-  cmake --build build
++    -DQT6=ON \
++    -DCMAKE_EXE_LINKER_FLAGS="-latomic"
```

Comment:
I suggest fixing this problem in upstream build system first.

Reply:
OK. Changes upstreamed in qbittorrent/qBittorrent#16788